docs/gadgets/snapshot/socket.md

```diff
@@ -5,6 +5,8 @@ description: >
   Gather information about TCP and UDP sockets.
 ---
 
+![Screencast of snapshot process compared to calling kubectl exec ps](capabilities.gif)
+
```

Comment:
What happened here? :laughing: Shouldn't it to include the `process.gif` in `process.md`?